.github/workflows/build-pdb.yml

```diff
@@ -30,3 +30,51 @@ jobs:
       
     - name: Build with Maven
       run: mvn -B clean install --file pom.xml -DskipTests
+
+  build-jdk11:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: Cache Maven packages
+        uses: actions/cache@v2.1.4
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}-jdk11
+          restore-keys: ${{ runner.os }}-m2-jdk11
+
+      - name: Build with Maven
+        run: mvn -B clean install --file pom.xml -DskipTests
```

Comment:
Can we use parallelism? `-T2C`

Reply:
This is a single module project. Parallelism doesn't make any difference.